kustomize/server-remote-repo-scan/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+resources:
+  - ../server-tls
+
+namespace: <TERRASCAN_NAMESPACE>
+
+configMapGenerator:
+  - name: ssh-known-hosts-config
+    files:
+      - known-hosts=.ssh/known_hosts
+
+secretGenerator:
+  - name: ssh-key-secret
+    files:
+      - private-key=.ssh/github2
+  - name: ssh-config-secret
```

Comment:
We do not need ssh config, if we are using a single key, this may create confusion for new users if they do not have a config file and are using a single key so it is better to remove ssh config